src/ResourceManager/Storage/Commands.Management.Storage/help/Update-AzureRmStorageAccountNetworkRuleSet.md

```diff
@@ -149,7 +149,7 @@ Accept wildcard characters: False
 ```
 
 ### -VirtualNetworkRule
-The Array of VirtualNetworkRule objects to update to the NetworkRule Property of a Storage Account.
+The Array of VirtualNetworkRule objects to update to the NetworkRule Property of a Storage account.
```

Comment:
Please revert the changes in the .psm1 files below this

Reply:
The *.psm1 files are removed.
How can I avoid these file generating in PowerShell developing?
Thanks!

Reply:
When you run Update-MarkdownHelp, or Update-MarkdownHelpModule, you can just specify a single cmdlet, or your own module